app/test_main.py

```diff
@@ -1,3 +1,57 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_value",
+    [
+        pytest.param(
+            0, 0,
+            [0, 0],
+            id="should return zero when dog-cat ages equal 0"
+        ),
+        pytest.param(
+            14, 14,
+            [0, 0],
+            id="should return zero when dog-cat ages less 14"
+        ),
```

Comment:
According to the task checklist, your tests shouldn't test the same case few times. Here, the cases with ages less than 15 are tested twice. It would be better to test only one case with ages less than 15.